src/Enzo/enzo_EnzoComputePressure.cpp

```diff
@@ -127,21 +135,24 @@ void EnzoComputePressure::compute_(Block * block,
       }
 
     } else {
-
       if (rank == 1) {
         for (int i=0; i<m; i++) {
-          p[i] = gm1 * d[i] *
-            (te[i] - 0.5*vx[i]*vx[i]);
+          enzo_float ke = 0.5*vx[i]*vx[i];
+          enzo_float me_den = mhd ? 0.5*bx[i]*bx[i] : 0.;
```

Comment:
I think you still need to include `by` and `bz` in the pressure in lower dimension studies.